docs/rm_to_eq_runner_payload_v2.md

```diff
@@ -88,7 +88,8 @@ The data property must adhere to one of [Business Survey Metadata][business_surv
 | **ru_name**          | The reporting unit’s display name                                                                                                                                             |
 | **trad_as**          | The reporting unit's 'trading as' name                                                                                                                                        |
 | **user_id**          | The id assigned by the respondent management system                                                                                                                           |
-| **survey_id**        | The survey identifier as used across the ONS.                                                                                                                                                                              |
+| **survey_id**        | The survey identifier as used across the ONS                                                                                                                                  |
+| **sds_dataset_id**   | The UUID of the dataset eQ runner will use to query SDS                                                                                                                       |
```

Comment:
By design can we confirm this is optional?

Reply:
Yes, this is optional, have added it to the doc